Source/JavaScriptCore/b3/B3ConstDoubleValue.cpp

```diff
@@ -140,14 +140,14 @@ Value* ConstDoubleValue::fMinConstant(Procedure& proc, const Value* other) const
 {
     if (!other->hasDouble())
         return nullptr;
-    return proc.add<ConstDoubleValue>(origin(), fMin(m_value, other->asDouble()));
+    return proc.add<ConstDoubleValue>(origin(), Math::fMin(m_value, other->asDouble()));
```

Comment:
This can be used from wasm, which can observe non-canonical NaNs, right?

Reply:
This is aligned to fmax behavior to ARM64, and no change is done in this patch.